.github/workflows/publish-npm.yml

```diff
@@ -26,27 +26,49 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: ${{ inputs.deploy-sha }}
+          fetch-tags: true
+          # fetch all history so that git describe works
+          fetch-depth: 0
```

Comment:
**Suggestion:** We can drop this if we're receiving the version as an input.

Let's receive the version in the input and use it here

Reply:
Check this pr for an implementation suggestion:

- https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/104/files

Reply:
Done! ✅